docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -35,70 +19,36 @@ All occurrences of the code before the `>>` must be substituted with the code af
 - `="conditionType" count="XOR"` >> `="conditionType" count="1"`
 - `="conditionType" count="-reset-XOR"` >> `="conditionType" count="-reset-1"`
 - `="isTwoHit" count="true"` >> `="hitPoints" count="2"`
-- `="isTwoHit" value="false"` >> `="hitPoints" value="1"`
-- `="isTwoHit" count="-reset-true"` >> `="hitPoints" count="-reset-2"`
 - `="isTwoHit" count="false"` >> `="hitPoints" count="1"`
+- `="isTwoHit" value="true"` >> `="hitPoints" value="2"`
+- `="isTwoHit" value="false"` >> `="hitPoints" count="1"`
+- `="isTwoHit" count="-reset-true"` >> `="hitPoints" count="-reset-2"`
 - `="isTwoHit" count="-reset-false"` >> `="hitPoints" count="-reset-1"`
 - `="occupiedTerrOf"` >> `="originalOwner"`
 - `="victoryCity" count="true"` >> `="victoryCity" count="1"`
-- `="victoryCity" count="-reset-true"` >> `="victoryCity" count="-reset-1"`
 - `="victoryCity" count="false"` >> `="victoryCity" count="0"`
+- `="victoryCity" value="true"` >> `="victoryCity" count="1"`
```

Comment:
This is wrong. You are changing "value" to "count".

Also, as per the above comment, I would list the "value" ones before the "count" ones (the value ones are more important as they act on the final attachments, while the other ones are for triggers changing them (rare case)).
This would be the listing order, with the (hopefully) correct code:
- `="victoryCity" value="true"` >> `="victoryCity" value="1"`
- `="victoryCity" value="false"` >> `="victoryCity" value="0"`
- `="victoryCity" count="true"` >> `="victoryCity" count="1"`
- `="victoryCity" count="false"` >> `="victoryCity" count="0"`
- `="victoryCity" count="-reset-true"` >> `="victoryCity" count="-reset-1"`
- `="victoryCity" count="-reset-false"` >> `="victoryCity" count="-reset-0"`

Reply:
Ugh, I didn't realize that VC option can be either value or count. IMO it should just be one or the other. Thanks for spotting this.

Updated in: c20682143

Reply:
They cannot, as options. However, generally speaking, when you use triggers to change options, you define the name as value and the value as count (so you cannot use the count but in its value-integrated form (having the count being the first element specified in "value", which, in this case, would be the first element specified in "count", since the "count" is the "value" (it actually doesn't matter in this specific case, as the "victoryCity" option happens not to have a "count" at all (you can specify a value, never a count, for it)))).

Since I feel what I just said is not very clear, I'll just redirect to the official information:
https://github.com/triplea-maps/the_pact_of_steel/blob/master/map/games/pact_of_steel_2.xml#L3397

As a warning, such "properties" have nothing to do with what is normally identified as "property", within the game XML (and of course, nothing to do with the other things identified as "properties" too outside of it).

The "value" ones are for the options, the "count" ones are for triggers that are changing such options during the course of the game (most likely, there is not a single game doing that, I guess).